docker.go

```diff
@@ -0,0 +1,44 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/loadimpact/k6/js/modules"
+)
+
+const version = "v0.0.2"
+
+func init() {
+	modules.Register("k6/x/docker", &Docker{
+		Version: version,
+	})
+}
+
+// Docker is the main export of k6 docker extension
+type Docker struct {
+	Version string
+}
+
+// Ps lists Docker containers
+func (d *Docker) Ps() []string {
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
+	}
+
+	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	result := make([]string, 0, len(containers))
+
+	for _, container := range containers {
+		result = append(result, fmt.Sprintf("%s %s %s", container.ID[:10], container.Image, container.Command))
```

Comment:
From what I understand, we're returning a string with all the values concatenated. Maybe it's better to return a struct?

So, it's easier to consume from JS.

Reply:
I agree with @dgzlopes on this one. If the struct itself is too big/contains a lot of properties we don't care about, I'd suggest remapping it to a bespoke struct with reasonable properties and proper json tags.

Reply:
@simskij  please take a look at [rework commit](https://github.com/k6io/xk6-docker/pull/1/commits/1b5d376742d53ab533cc510c60dc957eaee9d56e). I tried to keep the initial interface that came from docker go lib. Example js file shows how to get normal `docker ps` output now. IMO the benefits of this approach are transparent support of https://github.com/moby/moby/tree/master/client functionality. WDYT?

Reply:
I definitely agree!

If anything, I think it could end up being too much. I don't want to assume too much docker knowledge from the user. But let's start from here. If this extension takes off and people seem to be confused with how to use the props for some reason, we could always limit it in a future version.